src/goto-instrument/dump_c.cpp

```diff
@@ -857,6 +883,14 @@ bool dump_ct::ignore(const symbolt &symbol)
     system_headers.insert(it->second);
     return true;
   }
+  else if(!system_library_map.empty() &&
+          has_prefix(file_str, "/usr/include/") &&
```

Comment:
I suggest to declare a constant string, since it should be used consistently in both occurrences.